### PR TITLE
Change the DB design to avoid violating JSON syntax

### LINKED ISSUE #22: New way to document JSON database
The wrong JSON syntax prevents Sphinx from parsing it and doesn't highlight the keys, which annoys me.

### PR DESCRIPTION
**Pull request by Huy-Ngo**
In this request, I:

- Change the DB design documentation, resolve #22 
- Add some missing field 🤦‍♂ Somehow I missed creator for two objects

### CODE REVIEW

File: docs/source/solution/database.rst [hunk 1]
```diff
@@ -2,113 +2,83 @@ Database Design
...
-- user
-- project
-- task
-- discussion thread
-- comment
+- **`User`**
+- **`Project`**
+- **`Task`**
+- Discussion **`Thread`**
+- **`Comment`**
```
Comment (McSinyx) [inline docs/source/solution/database.rst:11]: FYI reStructuredText doesn't support nested inline markup.

File: docs/source/solution/database.rst [hunk 2]
```diff
@@ -2,113 +2,83 @@ Database Design
...
-      "name": "Foo Bar",
-      "email": "foo.bar@example.com",
-      "password-hash": "(hash generated for password)",
-      "role": "student",
-      "major": "ICT", // optional
-      "bio": "A self description" // optional
-   }
+Each **`User`** object has following attributes:
+
+- `id` (`string`): A UUID for the object
```
Comment (McSinyx) [inline docs/source/solution/database.rst:20]: [I gotta feeling](https://en.wikipedia.org/wiki/I_Gotta_Feeling) that [definition list](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#definition-lists) may better suits this.

**Review by McSinyx — COMMENTED** [review]
A few nits, otherwise LGTM.

**Review by McSinyx — DISMISSED** [review]

File: docs/source/solution/database.rst [hunk 3]
```diff
@@ -145,3 +146,17 @@ Each ``Comment`` object has following attributes:
...
+File
+----
+
+Each ``File`` object has following attributes:
+
+``id`` : ``string``
+   A UUID for the object
+
+``address`` : ``string``
+   The IPFS address for the file object.
```
Comment (McSinyx) [inline docs/source/solution/database.rst:159]: This is already the file's identity.

**Review by McSinyx — APPROVED** [review]

**Review by 9a24f0 — APPROVED** [review]

**Pull request merged**